domain/dcsa_ovs_domain.yaml

```diff
@@ -0,0 +1,294 @@
+# OpenAPI version identifier - required for OpenAPI 3.0 domains
+openapi: 3.0.0
```

Comment:
```suggestion
 openapi: 3.0.3
```

Reply:
Shouldn't we go for version 3.0.3 - some bugfixes might have been fixed ;-)

Reply:
changed